cdap-docs/admin-manual/source/_includes/installation/installation.txt

```diff
@@ -9,8 +9,11 @@ Preparing Package Managers
 CDAP components are available as either Yum ``.rpm`` or APT ``.deb`` packages. There is
 one package for each CDAP component, and each component may have multiple services.
 Additionally, there is a base CDAP package with three utility packages (for HBase
-compatibility) installed which creates the base configuration and the ``cdap`` user. We
-provide packages for *Ubuntu 12* and *CentOS 6*.
+compatibility) installed which creates the base configuration and the ``cdap`` user. 
+
+We provide packages for *Ubuntu 12* and *CentOS 6*. While these are the only packages we
+currently provide, they contain no OS-specific code, and the same packages will work on
```

Comment:
They do contain OS-specific code, even distribution-specific code. However, they do not contain distribution **version** specific code.